openedx/core/djangoapps/appsembler/api/tests/test_registration_api.py

```diff
@@ -116,11 +116,11 @@ def test_send_activation_email_without_password(self, send_activation_email):
         }
 
         def fake_send(user, profile, user_registration=None):
-            assert False, 'Should not call fake_send when no password'
+            assert True, 'Should not call fake_send when no password'
```

Comment:
`False` is intentional. Using `True` would effectively make the test case useless.

```suggestion
            assert False, 'Should not call fake_send when no password'
```